### PR TITLE
Multihash

### PR DESCRIPTION
**Pull request by dlongley**

### CODE REVIEW

**Comment by mattcollier** [comment]
Test suite is passing node v10.9.0.

File: lib/index.js [hunk 1]
```diff
@@ -586,8 +588,11 @@ api.consensus._hasher = (data, callback) => {
       format: 'application/n-quads'
     }, callback),
     hash: ['canonize', (results, callback) => {
-      const hash = niUri.digest('sha-256', results.canonize, true);
-      callback(null, hash);
+      const hash = crypto.createHash(
+        'sha256').update(results.canonize, 'utf8').digest();
```
Comment (davidlehn) [inline lib/index.js]: This split looks weird. Could break at the function call:
```js
      const hash = crypto.createHash('sha256')
￼        .update(results.canonize, 'utf8').digest();
```

File: test/package.json [hunk 2]
```diff
@@ -31,17 +31,18 @@
...
     "bedrock-ledger-validator-signature": "digitalbazaar/bedrock-ledger-validator-signature#master",
     "bedrock-mongodb": "^5.0.0",
     "bedrock-permission": "^2.3.2",
     "bedrock-test": "^2.0.0",
     "bedrock-validation": "^2.2.1",
     "grunt": "^1.0.1",
     "jsonld-signatures": "^2.1.4",
-    "ni-uri": "^1.1.3"
+    "multibase": "^0.4.0",
+    "multihashes": "^0.4.14"
```
Comment (davidlehn) [inline test/package.json]: These get pulled in as regular deps of bedrock-ledger-node.

**Comment by mattcollier** [comment]
@davidlehn your issues have been addressed.

**Comment by dlongley** [comment]
No one pull this in yet ... I might need to back these latest changes out because they were already done on the promises branch and it's going to get hairy.

**Comment by dlongley** [comment]
Ok, I've moved one of the two commits over to the promises branch (the other was already addressed in another way). I'll handle merging this soon.

**Pull request merged**